source/patterns/@aws-solutions-constructs/aws-lambda-kinesisstreams/test/integ.noArguments.ts

```diff
@@ -0,0 +1,31 @@
+/**
```

Comment:
How about at test with an existing VPC?

Reply:
Never mind, the unit tests around that seem pretty extensive

Reply:
yeah - there actually is an `integ.existingVpc` integration test also :)